addons/mrp/models/mrp_production.py

```diff
@@ -488,6 +494,8 @@ def write(self, vals):
                     raise UserError(_('You cannot move a planned manufacturing order.'))
             if 'move_raw_ids' in vals and production.state != 'draft':
                 production.move_raw_ids.filtered(lambda m: m.state == 'draft')._action_confirm()
+            if not production.routing_id and vals.get('date_planned_start') and not vals.get('date_planned_finished'):
```

Comment:
I would only do it if the current date_planned_finished is lower than the new date_planned_start

Reply:
``` python
if not production.routing_id and vals.get('date_planned_start') and not vals.get('date_planned_finished'):
    date_planned_start_dt = fields.Datetime.to_datetime(vals.get('date_planned_start'))
    if production.date_planned_finished < date_planned_start_dt:
        production.date_planned_finished = date_planned_start_dt + datetime.timedelta(hours=1)
```